docs/contrib/tutes/setup-linux.rst

```diff
@@ -11,6 +11,7 @@ Overview
 ========
 
 * Install Python 3.7
+* Install pip ``sudo apt-get install python3-pip``
```

Comment:
I think Python 3.7 comes with pip preinstalled when it is installed via pyenv, so I don't think we need this one.